### PR TITLE
feat: welcome message added to lending home page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Personalized welcome message added to lending home page

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
+					query: userInfoQuery,
+				});
 		},
+		result({ data }) {
+			this.userInfo = data?.my?.userAccount ?? {};
+		}
+	},
+	computed: {
+		firstName() {
+			return this.userInfo?.firstName ?? '';
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:40]: Do we need a fallback in the template if `firstName` is undefined, or can we expect it to be defined?

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -1,28 +1,44 @@
 <template>
-	<div class="tw-max-w-5xl tw-mx-auto">
-		New loan finding page
-	</div>
+	<www-page>
+		<div class="tw-max-w-5xl tw-mx-auto tw-p-2 lg:tw-pt-4">
+			<h3 class="tw-text-h3 tw-text-primary">
+				Welcome back, <span class="tw-text-action">{{ firstName }}</span>
```
[the remark this answers is not in the document]
Comment (mcstover) [reply src/pages/LoanFinding/LoanFinding.vue:5]: That's correct this needs to be masked.

[the remark this answers is not in the document]
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: masked

File: src/pages/LoanFinding/LoanFinding.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:40]: at this moment the page is for logged in users only, se we expect that firstName is always defined.
in some future sprints this page will become for logged out users too to consider this
thanks

**Review by dyersituations — APPROVED** [review]

**Pull request merged**